packages/cli/src/utils/generate.ts

```diff
@@ -76,6 +76,25 @@ function copyCoreFiles() {
   }
 }
 
+function copyPublicFiles() {
+  try {
+    if (existsSync(`${userDir}/public`)) {
+      copySync(`${userDir}/public`, `${tmpDir}/public`, {
+        overwrite: true,
+        filter: (src) => {
+          const allowList = ["json", "txt", "public"]
```

Comment:
This might be a small issue, but we're defining this as a const inside a loop, `filter` is being called on every file, right? Should we define `allowList` outside this scope? Maybe inside `copyPublicFiles`?

Reply:
Sure, done!